modules/pam_time/pam_time.c

```diff
@@ -79,6 +82,19 @@ _pam_parse (const pam_handle_t *pamh, int argc, const char **argv, const char **
 	}
     }
 
+#ifdef VENDOR_PAM_TIME_CONF
+    if (*conffile == PAM_TIME_CONF) {
+      /* Check whether PAM_TIME_CONF file is available.
+       * If it does not exist, fall back to VENDOR_PAM_TIME_CONF file. */
+      struct stat buffer;
+      if (stat(*conffile, &buffer) != 0 && errno == ENOENT) {
+	*conffile = VENDOR_PAM_TIME_CONF;
+	pam_syslog (pamh, LOG_INFO,
+		    "cannot read settings from %s. Trying %s", PAM_TIME_CONF, *conffile);
```

Comment:
Note that `pam_sm_acct_mgmt` already does some logging:
```
   ctrl = _pam_parse(pamh, argc, argv, &conf_file);

    if (ctrl & PAM_DEBUG_ARG) {
        pam_syslog(pamh, LOG_DEBUG, "conffile=%s", conf_file);
    }
```

Why do you suggest to log this one more time in `_pam_parse` with a higher level of importance?